src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -124,6 +191,23 @@ public static void RemoveReadOnlyThrowsWhenFound()
             }
         }
 
+        /* Placeholder information for these tests until they can be written to run reliably.
+         * Currently such tests would create physical files (Unix) and\or certificates (Windows)
+         * which can collide with other running tests that use the same cert, or from a
+         * test suite running more than once at the same time on the same machine.
+         * Ideally, we have lower testing hooks or use Microsoft Fakes Framework to redirect
+         * and encapsulate the actual storage logic so it can be tested.
+         * See issue https://github.com/dotnet/corefx/issues/12833
```

Comment:
This comment is a bit inaccurate.

The tests could use a new store, GUID-ly named, like the X509FileSystemTests.Unix.cs-housed tests.  The problem is that on Windows there's not an easy (non P/Invoke) way to delete a cert store.

The right approach, it seems to me, is to make a P/Invoking store deleter helper type for Windows (and reuse the one for Unix), and then Add/Remove/AddRange/RemoveRange tests can be written.  Since they'd be modifying state (and therefore dirty a machine during abnormal termination) they'd all (continue to) be OuterLoop.  And that's what I'm planning on doing for #12223; since we don't actually have any tests for Add or Remove on Windows, currently.  We just run on faith that the code'll keep working.